implementation-details.md

```diff
@@ -22,16 +22,26 @@ Each Solana node participates in gossip in one of two modes:
 
 ## Connecting to the cluster
 
-When connecting a node to the cluster, the node operator must specify an entrypoint. The entrypoint is the gossip address of the peer the node is going to initially connect to. The node starts up and then advertises itself to the cluster by sending messages to the peer identified by the entrypoint. The node sends:
-* two [push messages](#push-messages) containing its own [`Version`](gossip-protocol-spec.md#version) and [`NodeInstance`](gossip-protocol-spec.md#nodeinstance) 
-* a [pull request](#pull-requests) that contains the node's own [`LegacyContactInfo`](gossip-protocol-spec.md#legacycontactinfo)
+When connecting a node to the cluster, the node operator must specify an entrypoint. The entrypoint is the gossip address of the peer the node will initially connect to.
```

Comment:
> When connecting a node to the cluster, the node operator must specify an entrypoint.

When connecting a node to the cluster, the node operator must specify at least one entrypoint.